auth-service/src/test/kotlin/uk/gov/communities/delta/security/RegistrationServiceTest.kt

```diff
@@ -48,7 +48,7 @@ class RegistrationServiceTest {
     fun testRegisteringNewStandardUser() = testSuspend {
         val registrationResult = registrationService.register(
             Registration("Test", "User", "user@example.com"),
-            listOf(Organisation(orgCode)),
+            listOf(Organisation(orgCode, name = "Test org")),
```

Comment:
Are you able to skip  `name =` if the optional parameter (retirement date) is moved to the end? 

And just slightly inconsistent - omitting the optional param here vs passing in null in OAuthSSOLoginTest.

Reply:
Makes sense, done